py3status/modules/dropboxd_status.py

```diff
@@ -43,8 +43,8 @@
 {'color': '#FF0000', 'full_text': "Dropbox: isn't running"}
 """
 
-STRING_UNAVAILABLE = "Dropbox: isn't installed"
 STRING_ERROR = "Dropbox: command failed"
+STRING_NOT_INSTALLED = "Dropbox: isn't installed"
```

Comment:
will the user see `dropbox: Dropbox: isn't installed`

Reply:
Good catch. Fixed.